Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs

```diff
@@ -83,6 +83,7 @@ public void onSoomlaProfileInitialized()
 		{
 			SoomlaUtils.LogDebug(TAG, "SOOMLA/UNITY onSoomlaProfileInitialized");
 
+			SoomlaProfile.nativeModulesInitialized = true;
```

Comment:
`onProfileInitialized` event (staring not from capital letter) invokes from native part, so we can determine completion of native initialization using this point.